docs/source/release_roadmap.org

```diff
@@ -0,0 +1,363 @@
+#+TITLE: Recommended Road
+#+AUTHOR: Aleksandr Petrosyan
+#+DATE: 2023-08-13
+
+* State of the project
+
+Iroha v2 is a capable ledger with notable implemented subroutines that
+allow it to be deployed successfully as a private blockchain ledger:
```

Comment:
I have a feeling that some points sounds a bit subjective. It will be better if we can provide some quantitative measurements here.
Like "good performance characteristics" compared to what?
Or "is a capable ledger" does it have feature parity with it's competitors in the private blockchain context?

Reply:
Good point.